app/views/hiring_staff/vacancies/edit_vacancy_sections/_edit_application_details.html.haml

```diff
@@ -18,7 +18,7 @@
 
   .app-check-your-answers__contents
     %dt.app-check-your-answers__question#school_visits
-      %h4.govuk-heading-s= t("jobs.#{organisation}")
+      %h4.govuk-heading-s= t("jobs.#{organisation_type_basic(@vacancy.school_or_school_group)}_visits")
```

Comment:
Doesn't this depend on where the job is located as opposed to whether the organisation is a Trust or not?

Reply:
Yeah, I believe that is why the organisation provided to organisation_type_basic comes from the vacancy. If the vacancy is at the school, @vacancy.school_or_school_group will return the school, if not, it will return the school group... I think?

Reply:
Yep, my bad sorry!